scheduler/src/Rsbc.Dmf.Scheduler/Startup.cs

```diff
@@ -349,8 +321,15 @@ public void ConfigureServices(IServiceCollection services)
                 services.AddTransient(_ => new BcMailAdapterClient(channel));
 
             }
+
+            if (Configuration["SCHEDULER_SERVICE_URI"] != null)
```

Comment:
I don't think we need this condition here.  Can you change it so the transient is always added.